resources/RegexGrammar.pp

```diff
@@ -42,11 +42,13 @@
 //
 
 // Character classes.
+%token  negative_class_fc_       \[\^(?=\])         -> class_fc // same as "negative_class_" but followed by "class:_class"
+%token  class_fc_                \[(?=\])           -> class_fc // same as "class_"          but followed by "class:_class"
+%token class_fc:_class           \]                 -> class
```

Comment:
Is this really needed? Can you give me an example of what it fixes? I was under the impression it worked fine with `class:_class_literal` before, but maybe I misunderstand something.

Reply:
yes - https://phpstan.org/r/7ed7c54b-ec78-4165-a8d2-857d3dbbb14e

Reply:
Ah I see. Makes sense thanks 👍🏻